setup.py

```diff
@@ -11,6 +11,7 @@
     'requests',
     'pyparsing',
     'scikit-learn',
+    'pytorch-lightning>=1.6',
```

Comment:
Let's move this to `full_install_requires`. In addition, is the `1.6` version requirement necessary? PyG+PL works fine for version 1.5.* as well.

Reply:
Sure, It should work well with 1.5* as well.

Reply:
> Let's move this to full_install_requires

This will affect the `Minimal Installation` test since it checks for successful import of `torch_geometric.graphgym` module. Now that we are integrating PL for GraphGym shall we remove this from the minimal installation test?

Reply:
Sounds good. Let's remove the import from the test until we make some further progress :)